src/components/Header/Header.js

```diff
@@ -37,31 +41,43 @@ export const Header = () => {
     }
 
     return (
-        <Navbar className='navbar-fixed p-navbar' bg="light" expand="lg">
+        <div className="navbar-fixed">
+        <Navbar className='p-navbar' bg="white" expand="lg">
             <LinkContainer to='/'>
                 <Navbar.Brand className="navbar-title">Bedu Store</Navbar.Brand>
             </LinkContainer>
             <Navbar.Toggle aria-controls="basic-navbar-nav"/>
             <Navbar.Collapse id="basic-navbar-nav">
-                <Nav className="m-auto">
-                    <LinkContainer to='/store'>
-                        <Nav.Link className="navbar-links">
-                            Productos
-                        </Nav.Link>
-                    </LinkContainer>
-                </Nav>
-                <div className="navbar-right d-flex justify-content-around align-items-center">
+                <Nav className="w-80 py-2 my-md-0">
                     <SearchInput 
                         value={ searchValue } 
                         onChange={ setSearchValue } 
                         onSubmit={ searchProduct } 
                     />
+                </Nav>
+                <div className="d-flex justify-content-around">
                     <Link className='text-dark mx-4' to='/cart'>
                         {
                             cartItems.length > 0 && <Badge className='bg-color-orange text-white' pill>{cartItems.length}</Badge>
                         }
                         <FaShoppingCart size={20}/>    
                     </Link>
+                    { 
+                     !userInfo ?
+                        <div>
+                        <Link to='/login' className="pr-2 border-right text-dark font-weight-bolder">
+                            Iniciar sesión
+                        </Link>
+                        <Link to='/register' className="ml-2 text-dark font-weight-bolder">
+                            Registrarse
+                        </Link>
+                        </div>
+                        : ''
```

Comment:
Dar tab al contenido del div para indentarlo un espacio adentro y sugeriria cambiar el ternario por un and (&&) ya no que se esta regresando una cadena vacía en el else.

`!userInfo &&
                        <div>
                            <Link to='/login' className="pr-2 border-right text-dark font-weight-bolder">
                                Iniciar sesión
                            </Link>
                            <Link to='/register' className="ml-2 text-dark font-weight-bolder">
                                Registrarse
                            </Link>
                        </div>